docs/github-secrets-quickstart.md

```
@@ -32,7 +32,11 @@ gh api --method PUT /repos/POWERFULMOVES/PMOVES.AI/environments/Prod \
 ```
 
 Or via GitHub UI:
+<<<<<<< HEAD
+1. Go to [GitHub Environments Settings](https://github.com/POWERFULMOVES/PMOVES.AI/settings/environments)
+=======
 1. Go to https://github.com/POWERFULMOVES/PMOVES.AI/settings/environments
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)